pkg/commands/process/settings/rules/javascript/lang/logger.yml

```diff
@@ -1,4 +1,4 @@
-type: "risk"
+type: risk
```

Comment:
Accidental change? (and following one)

Reply:
it's intentional, `"` are redundant here, @cfabianski and me were removing them on some call so figured it's a standard we want to introduce